internal/messaging/messaging.go

```diff
@@ -107,6 +107,9 @@ func BootstrapHandler(ctx context.Context, wg *sync.WaitGroup, startupTimer star
 				messageBusInfo.PublishTopicPrefix,
 				messageBusInfo.AuthMode))
 
+			// Make sure the MessageBus password is not leaked into the Service Config that can be retrieved via the /config endpoint
+			delete(messageBusInfo.Optional, bootstrapMessaging.OptionsPasswordKey)
```

Comment:
This is not needed. IOTech addressed the issue in a different slightly better way.
https://github.com/edgexfoundry/device-sdk-go/commit/77f757bcec6df927aad1820bf5eaf41e09e0f5b2